notes/README.md

```diff
@@ -0,0 +1,60 @@
+
+Notes
+===========
```

Comment:
Use `# Notes` to follow other headings.

Reply:
Done. All changes are addressed, I believe?